spec/controllers/api/api_controller_spec.rb

```diff
@@ -3,7 +3,7 @@
 require "rails_helper"
 
 class DummyController < Api::ApiController
-  include Api::ApiTokenAuthenticatable
+  include ApiTokenAuthenticatable
 end
```

Comment:
Why this change?

Reply:
It didn't match the actual namespace since it appears in the `concerns` directory which is treated as a root namespace by zeitwerk